docs/developer-guide/services.md

```diff
@@ -223,10 +223,10 @@ Each Prowler service requires a service client to use the service in the checks.
 The following is the `<new_service_name>_client.py` containing the initialization of the service's class we have just created so the service's checks can use them:
 
 ```python
-from prowler.providers.<provider>.lib.audit_info.audit_info import audit_info
+from prowler.providers.common.common.get_global_provider import set_mocked_<provider>_provider
```

Comment:
I think this path is not right, please check it out.

Reply:
Checked ✅ btw you were right